enshrine-esdt-safe/src/from_sovereign/transfer_tokens.rs

```diff
@@ -287,18 +285,18 @@ pub trait TransferTokensModule:
         }
 
         // deposit back mainchain tokens into user account
-        let sc_address = self.blockchain().get_sc_address();
-        let tx_nonce = self.get_and_save_next_tx_id();
-
-        self.deposit_event(
-            &operation_tuple.operation.data.op_sender,
-            &operation_tuple.operation.get_tokens_as_tuple_arr(),
-            OperationData {
-                op_nonce: tx_nonce,
-                op_sender: sc_address.clone(),
-                opt_transfer_data: None,
-            },
-        );
+        let _sc_address = self.blockchain().get_sc_address();
+        let _tx_nonce = self.get_and_save_next_tx_id();
+
+        // self.deposit_event(
```

Comment:
commented code

Reply:
this code will be changed once the the `feat/enshrine` branch will be merged into this one, the commented code is a really old version that will be replaced